pkg/monitor/monitor.go

```diff
@@ -27,7 +26,7 @@ type RuntimeConfig struct {
 	LBConfig *config.ApiLBConfig
 }
 
-func Monitor(clusterName, clusterDomain, templatePath, cfgPath, apiVip string, apiPort, lbPort, statPort uint16, interval time.Duration) error {
+func Monitor(kubeconfigPath, clusterName, clusterDomain, templatePath, cfgPath, apiVip string, apiPort, lbPort, statPort uint16, interval time.Duration) error {
```

Comment:
Yep, it's an API change, but we should be safe with this change since we have only internal consumers (well it's a single consumer: cmd/monitor) and we updated both consumer and provider.